examples/java/testcontainer/src/test/java/com/grafana/example/TestcontainerTest.java

```diff
@@ -43,25 +48,63 @@ void testExportMetric() {
     var app = new OtelApp();
     app.run();
 
+    HttpClient client = HttpClient.newHttpClient();
     String query =
         URLEncoder.encode(
-            "sold_items_total{job=\"otel-java-test\",service_name=\"otel-java-test\",tenant=\"tenant1\"}",
```

Comment:
for some reason this `service_name` tag did not work in any of my testing. It was never available as a label when querying and removing it was the only way for me to get the test to pass. I would have thought [this config](https://github.com/grafana/docker-otel-lgtm/blob/0fa1021cca8745b471bf3027512237528049409c/docker/prometheus.yaml#L7) would ensure this was populated, but I must be missing something.

Reply:
yes, that's the right config
maybe you're not using the lastet image - try setting the tag explicitly

Reply:
just tried locally - it's working for me